.github/workflows/close-stale-issues.yml

```diff
@@ -0,0 +1,18 @@
+name: "Close stale issues"
+on:
+  schedule:
+    - cron: "12 3 * * *" # arbitrary time not to DDOS GitHub
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'This issue was marked stale due to lack of activity. It will be closed in 30 days.'
+        close-issue-message: 'Closed as inactive. Feel free to reopen if this issue needs resolving.'
+        exempt-issue-labels: 'feature-request'
```

Comment:
should we add something like "triaged" to represent bugs that have been acknowledged by the maintainers and where no more info is required from the OP? I think there might be bug reports that are clear and we intent to solve but don't have to bandwidth to so. Might be good to have a label for such issues.

Reply:
added "triaged"